react-components/src/architecture/base/utilities/geometry/getBoundingBoxFromPlanes.test.ts

```diff
@@ -0,0 +1,84 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import { describe, expect, test } from 'vitest';
+
+import { Vector3, type Plane } from 'three';
+import { getBoundingBoxFromPlanes } from './getBoundingBoxFromPlanes';
+import { Range3 } from './Range3';
+import { createPlanesAtTheEdgesOfBox } from './isBoxVisibleByPlanes.test';
+
+const min = new Vector3(100, 200, 300);
+const max = new Vector3(400, 600, 800);
+const originalBox = new Range3(min, max);
+
+describe('getBoundingBoxFromPlanes', () => {
```

Comment:
we are starting using the title with the function name prop as a pattern for unit test description titles:
```suggestion
describe(getBoundingBoxFromPlanes.name, () => {
```